### DISCUSSION TITLE
The rich editor's toolbar is unusable with very large contents

### DISCUSSION THREAD

**Discussion by Rattone** [filamentphp/filament#11685] [Ideas]
Because it's not sticky and the content increases the size of the text area.
The same problem affects the markdown editor.

**Comment by meharenderb** [comment]
I implemented this solution, and it worked perfectly without modifying any core CSS of Filament:

`
->extraInputAttributes(['class' => 'max-h-96', 'style' => 'overflow-y: scroll;'])
`

**Reply by maurozadu** [reply]
not workiing in v4

**Comment by matildasmets** [comment]
Use the floating toolbar
https://filamentphp.com/docs/4.x/forms/rich-editor#customizing-floating-toolbars
